pkg/config/defaults_darwin.go

```diff
@@ -20,15 +21,24 @@ const (
 	fallbackCPUs   int     = 2
 )
 
-func vmDefault(cfg *Finch) {
+func vmDefault(cfg *Finch, supportsVz bool) {
 	if cfg.VMType == nil {
-		cfg.VMType = pointer.String("qemu")
+		if supportsVz {
+			cfg.VMType = pointer.String("vz")
+		} else {
+			cfg.VMType = pointer.String("qemu")
+		}
 	}
 }
 
-func rosettaDefault(cfg *Finch) {
+func rosettaDefault(cfg *Finch, supportsVz bool) {
```

Comment:
are we saying if vz then rosetta is true by default?

Reply:
Yep I think most people want this behavior since Rosetta is faster

Reply:
Oh, I missed that. Thanks Jan. I'll make a revision that disables Rosetta by default, and include that link in the docs